modules/business-text/business-text.service.js

```diff
@@ -26,16 +26,15 @@ class BusinessTextService {
   }
 
   async updateBusinessText(id, businessText) {
-    if (await this.checkBusinessTextExist(businessText, id)) {
```

Comment:
Why have you removed this check? I think it is required here

Reply:
`check Business Text Exist` looking for existing with a same code so it is not necessary. If the page is not found - the error code will return below.